### ISSUE TITLE
Адаптация к ssd/Adaptation to ssd

### ISSUE THREAD

**Issue by Chrome2574** [bleachbit/wishlist#126]
Russian version:
Здравствуйте, уважаемый разработчик! Пожалуйста адаптируете bleachBit к ssd подключаемым по usb и накопителям самого устройства. Всё же в ssd другой принцип работы.
 English version:
Hello, dear developer! Please adapt bleachBit to ssd connected via usb and storage devices of the device itself. However, ssds have a different operating principle.

**Comment by az0** [comment]
Are you asking for BleachBit to clean a storage device with direct disk access at a low level when the file system is not mounted?

**Comment by Chrome2574** [comment]
No, I ask that bleachbit work correctly with ssd on exfat, fat32 and ntfs file systems

**Comment by Chrome2574** [comment]
Russian version:
Хорошо,я не так задал вопрос.,пожалуйста,BleachBit 
работает с SSD и SSD m2?
Просто эти накопители сейчас выходят 
на первый план и будет обидно,
если BleachBit потеряет актуальность.
English version:
Okay, I asked the question wrong, please, BleachBit
works with SSD and SSD m2?
It's just that these drives are coming out now
to the forefront and it will be a shame,
if BleachBit becomes irrelevant.

**Comment by az0** [comment]
> BleachBit works with SSD and SSD m2?

Yes, BleachBit works with solid state drives, including M2 SSDs.

**Comment by Chrome2574** [comment]
Thank you for your reply
--
Отправлено из Mail.ru для Android пятница, 29 марта 2024г., 17:57 +03:00 от Andrew Ziem  ***@***.*** :
